### ISSUE TITLE
Desktop View: Member Profile

### ISSUE THREAD

**Issue by mk4111** [emfoundation/ce100-app#634]
Need to edit view of Member Profile page for Desktop use.

![screenshot from 2017-01-24 18-32-48](https://cloud.githubusercontent.com/assets/13470325/22261095/89b74bcc-e263-11e6-8c32-7ce5f5022f36.png)

![screenshot from 2017-01-24 18-32-30](https://cloud.githubusercontent.com/assets/13470325/22261097/8ba80548-e263-11e6-8b0e-26b6201ca973.png)

**Comment by mk4111** [comment]
Update - current issues:
- mission statement/tags are in different cards for mobile; need to be combined for desktop view --> morning standup: leaving them as cards is fine for now
- Challenges card for 'Add New Challenge' does not exist in mobile view
---------------------------------------
- Priority for this view: get columns

**Comment by asimonw** [comment]
@fannyBkane Could you have a look at this? LGTM, but the transition to tablet (and tablet style) isn't there yet. We could cover that separately in the next sprint though.

**Comment by asimonw** [comment]
@mk4111 Two small issues:
* Could the background colour on desktop stretch all the way to the bottom, even when there isn't enough content to fill the whole space above the fold?
* The challenge icons don't look good on mobile (see below). I think they are there because clicking a challenge was supposed to open a challenge. Since we'll go for a non-modal option for now, maybe simply comment out the icons? @fannyBkane do you agree with that?
<img width="357" alt="screen shot 2017-02-24 at 12 30 02" src="https://cloud.githubusercontent.com/assets/15235804/23303707/901667cc-fa8d-11e6-82a0-75639a28f16e.png">

**Comment by fannyBkane** [comment]
@asimonw @mk4111 it's fine, thanks guys!

**Comment by JMurphyWeb** [comment]
@asimonw so for now we don't need the icons in mobile or desktop?

**Comment by asimonw** [comment]
@JMurphyWeb Yes indeed. We might bring them back once we have a modal solution.

**Comment by mk4111** [comment]
Fixing both issues raised by @asimonw [above](https://github.com/emfoundation/ce100-app/issues/634#issuecomment-282281294) in #732

**Comment by mk4111** [comment]
Coming towards the end of my pomodoro, so updating:
- Have tried to set html to `height: 100%`, and body to `min-height: `100%` but with no effect

**Comment by mk4111** [comment]
Going to park this issue for now - will revisit later today once I've had an external set of eyes to have a look too!

**Comment by mk4111** [comment]
Just fixed this with a change in container padding and height + html/body height.

Need to check with other screens but this current implementation fixes the background colour issue raised.

![screenshot from 2017-02-28 12-19-25](https://cloud.githubusercontent.com/assets/13470325/23405050/2d01520c-fdb0-11e6-957e-eeee74886b9c.png)
